src/command/ig.jl

```diff
@@ -11,7 +9,7 @@ function commander(c::Client, m::Message, ::Val{:ig})
 
     if length(args) == 0 ||
             args[1] ∉ ["start-game", "abandon-game", "quote", "chart",
-                "buy", "sell", "rank", "view",
+                "buy", "sell", "perf", "rank", "view",
```

Comment:
Wouldn't `hist` be better than `perf`?

Reply:
Nevermind. I guess "performance" is a usual technical term for that, right?

Reply:
Performance generally refers to the measurement of profit/loss of some investment. The current implementation contains calculation for a 1-day performance. We could possibly implement more options as you suggested on Discord to allow calculations of `n`-day performance as well. However, it's a little off track from a portfolio management perspective because you may not hold the same stocks n-days ago.